web/components/atomic/molecules/Modal/Modal.stories.tsx

```diff
@@ -3,12 +3,12 @@ import { ComponentStory, ComponentMeta } from '@storybook/react';
 import { Modal } from './Modal';
 
 export default {
-  title: 'owncast/Modals/Container',
+  title: 'molecules/Modal',
   component: Modal,
   parameters: {
     docs: {
       description: {
-        component: `This is the popup modal container that all modal content is rendered inside. It can be passed content nodes to render, or a URL to show an iframe.`,
+        component: `This is the container that all modal content is rendered inside. It can be passed content nodes to render, or a URL to show an iframe.`,
       },
     },
   },
```

Comment:
(GH won't let me add a suggestion 😐)

Suggestion: Remove these lines and copy the description text as a comment to the component in JSDoc syntax. This will allow IDEs to find the component's description, and enables further tooling in the future.